File: dart-jaguar/lib/model/api_key_response_all_of.dart
import 'package:jaguar_serializer/jaguar_serializer.dart';


part 'api_key_response_all_of.jser.dart';

class ApiKeyResponseAllOf {
  
  @Alias('result', isNullable: true,  )
  final Object result;
  
  @Alias('msg', isNullable: true,  )
  final Object msg;
   /* The API key that can be used to authenticate as the requested user.  */
  @Alias('api_key', isNullable: false,  )
  final String apiKey;
   /* The email address of the user who owns the API key  */
  @Alias('email', isNullable: false,  )
  final String email;
  

  ApiKeyResponseAllOf(
      

{
     this.result = null,  
     this.msg = null,  
    
     this.apiKey = null,  
     this.email = null 
    }
  );

  @override
  String toString() {
    return 'ApiKeyResponseAllOf[result=$result, msg=$msg, apiKey=$apiKey, email=$email, ]';
  }
}

@GenSerializer(nullableFields: true)
class ApiKeyResponseAllOfSerializer extends Serializer<ApiKeyResponseAllOf> with _$ApiKeyResponseAllOfSerializer {

}

File: scala-play-server/app/model/ApiKeyResponseAllOf.scala
package model

import play.api.libs.json._

/**
  * Represents the Swagger definition for ApiKeyResponse_allOf.
  * @param apiKey The API key that can be used to authenticate as the requested user. 
  * @param email The email address of the user who owns the API key 
  */
@javax.annotation.Generated(value = Array("org.openapitools.codegen.languages.ScalaPlayFrameworkServerCodegen"), date = "2021-08-09T21:37:15.417726Z[Etc/UTC]")
case class ApiKeyResponseAllOf(
  result: Option[OasAnyTypeNotMapped],
  msg: Option[OasAnyTypeNotMapped],
  apiKey: String,
  email: String
)

object ApiKeyResponseAllOf {
  implicit lazy val apiKeyResponseAllOfJsonFormat: Format[ApiKeyResponseAllOf] = Json.format[ApiKeyResponseAllOf]
}

